src/views/Arys/ActionButtons/AryCopyModal/index.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { useState, useCallback } from 'react';
+import { connect } from 'react-redux';
+
+import Modal from '#rscv/Modal';
+import ModalHeader from '#rscv/Modal/Header';
+import ModalBody from '#rscv/Modal/Body';
+import ModalFooter from '#rscv/Modal/Footer';
+import ChecklistInput from '#rsci/ChecklistInput';
+import SearchInput from '#rsci/SearchInput';
+
+import Button from '#rsu/../v2/Action/Button';
+
+import {
+    currentUserAdminProjectsSelector,
+} from '#redux';
+import {
+    RequestClient,
+    methods,
+} from '#request';
+
+import notify from '#notify';
+import _ts from '#ts';
+import _cs from '#cs';
+
+import {
+    AddRequestProps,
+    Requests,
+    AppState,
+} from '#typings';
+
+import styles from './styles.scss';
+
+interface OwnProps {
+    className?: string;
+    requests: () => void;
+    closeModal?: () => void;
+    userProjects: string[];
+}
+
+interface ProjectItem {
+    id: number;
+    title: string;
+}
+
+interface PropsFromAppState {
+    userProjects: ProjectItem[];
+}
+
+interface Body {
+    assessments: number[];
+    projects: number[];
+}
+
+interface Params {
+    bodyToSendToApi: Body;
+}
+
+type ComponentPropsWithAppState = PropsFromAppState & OwnProps;
+
+const mapStateToProps = (state: AppState) => ({
+    userProjects: currentUserAdminProjectsSelector(state),
+});
+
+const requestOptions: Requests<ComponentPropsWithAppState, Params> = {
+    aryCopyRequest: {
+        url: '/assessment-copy/',
+        body: ({ params: { bodyToSendToApi } }) => bodyToSendToApi,
+        method: methods.POST,
+        onSuccess: ({
+            response,
+            props: {
+                closeModal,
+            },
+        }) => {
+            console.warn('response', response);
+            const message = _ts(
+                'assessments.copyModal',
+                'successNotify',
+                { projectsCount: response.projects.length },
```

Comment:
check if the `projects` exist before returning the length

`projectsCount: response.projects ? response.projects.length : 0